### PR TITLE
Run yum update in final container

### PR DESCRIPTION
**Pull request by JohnStrunk**
**Describe what this PR does**
This moves the final image of the containers to centos:7 and adds a `yum
update` to the container build process in order to make sure we
incorporate the latest security fixes.

**Is there anything that requires special attention?**
no

**Related issues:**

### CODE REVIEW

File: pkg/gluster-virtblock/Dockerfile [hunk 1]
```diff
@@ -58,10 +58,11 @@ RUN ldd /build/glustervirtblock-csi-driver | grep -q "not a dynamic executable"
 
 #-- Final container
 
-FROM docker.io/centos:7.5.1804 as final
+FROM docker.io/centos:7 as final
 
 # Install dependencies
-RUN yum -y install centos-release-gluster && \
+RUN yum update -y && \
+    yum -y install centos-release-gluster && \
```
Comment (Madhu-1) [inline pkg/gluster-virtblock/Dockerfile]: @JohnStrunkI think we are missing `ADD http://artifacts.ci.centos.org/gluster/nightly/master.repo  /etc/yum.repos.d/glusterfs-nightly.repo` in this dockerfile  can you  add  it?

Comment (JohnStrunk) [reply pkg/gluster-virtblock/Dockerfile]: done

Comment (Madhu-1) [reply pkg/gluster-virtblock/Dockerfile]: @JohnStrunk do we need `centos-release-gluster` or `glusterfs-fuse` here?

Comment (Madhu-1) [reply pkg/gluster-virtblock/Dockerfile]: @kotreshhr why `centos-release-gluster` is added here?

Comment (JohnStrunk) [reply pkg/gluster-virtblock/Dockerfile]: We probably just need the fuse driver like the glusterfs driver container

**Comment by nixpanic** [comment]
On Mon, Feb 04, 2019 at 08:48:02AM -0800, Madhu Rajanna wrote:
> Madhu-1 commented on this pull request.
> 
> 
> 
> >  
>  # Install dependencies
> -RUN yum -y install centos-release-gluster && \
> +RUN yum update -y && \
> +    yum -y install centos-release-gluster && \
> 
> @kotreshhr why `centos-release-gluster` is added here?

The centos-release-gluster package provides the YUM repository that
contains the glusterfs-fuse packages which is installed in one of the
next yum commands.

**Comment by JohnStrunk** [comment]
> The centos-release-gluster package provides the YUM repository that contains the glusterfs-fuse packages which is installed in one of the next yum commands.

@nixpanic Since we're manually adding the nightly repo, that means we can remove this. correct?

**Comment by nixpanic** [comment]
On Mon, Feb 04, 2019 at 05:06:04PM +0000, John Strunk wrote:
> > The centos-release-gluster package provides the YUM repository that contains the glusterfs-fuse packages which is installed in one of the next yum commands.
> 
> @nixpanic Since we're manually adding the nightly repo, that means we can remove this. correct?
> 

There might be other dependencies that are not part of the nightly
repositories. At the moment I do not think glusterfs-fuse depends on
anything extra, but glusterfs-server definitely does. It is possible
that in the future glusterfs-fuse will also require userspace-rcu, and
the version of the library will need to match what was used for building
(the CentOS Storage SIG one, not the older EPEL package).

So at the moment it is probably fine to drop it, but it might need to
get added in the future again.

**Comment by JohnStrunk** [comment]
Thanks.

@Madhu-1 I think this is ready.

**Pull request merged**